### PR TITLE
env: add version dependent config for s3 gw

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -237,7 +237,7 @@ jobs:
         env:
           ALLURE_RESULTS_DIR: ${{ env.ALLURE_RESULTS_DIR }}
         run: |
-          source venv.pytest/bin/activate && pytest --alluredir=${GITHUB_WORKSPACE}/allure-results pytest_tests/tests
+          source venv.pytest/bin/activate && pytest --log-cli-level error --alluredir=${GITHUB_WORKSPACE}/allure-results pytest_tests/tests
```
Comment (roman-khimov) [inline .github/workflows/run-tests.yml:240]: Why?

Comment (evgeniiz321) [reply .github/workflows/run-tests.yml:240]: We don't need all of the logs when running in CI. They make it harder to see actual errors. Besides, we regularly have allure, that makes these logs even more useless in CI. When running locally - it is a different story and the log level is INFO there.

Comment (roman-khimov) [reply .github/workflows/run-tests.yml:240]: Why is it not explained in the commit message?

Comment (evgeniiz321) [reply .github/workflows/run-tests.yml:240]: Updated the commit message.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**